opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/indicator-test.ts

```diff
@@ -243,6 +245,16 @@ describe('Indicator resolver standard behavior', () => {
     });
     expect(queryResult.data?.indicatorFieldPatch.name).toEqual('Indicator - test');
   });
+  it('should update indicator pattern, causing change in stix id', async () => {
```

Comment:
how this test is related to pattern upsert ? updating the pattern was already changing the standard_id, no ?

Reply:
or does it impact the next tests about sync maybe ?

Reply:
yes it's about sync issues. The standard id changes on one side but not on the other.